packages/components/psammead-most-read/README.md

```diff
@@ -71,6 +72,24 @@ const items = [
 />;
 ```
 
+The `className` prop allows additional styles to be applied to the components `MostReadSection` wrapper. This is intended to be passed by `styled-components`.
+
+```jsx
+import styled from 'styled-components';
```

Comment:
Sorry for the misunderstanding, but I meant to give me an example of how the `className` is passed and used, not to add it to the README. 

Still, I can not understand how the `className` is used in this example. This just shows how to override the styles defined in MostRead using styled-components. Is that what you are trying to achieve?

Reply:
Hmm yes the intention is that it's never used directly, styled-components will pass a className in this case. There's nothing stopping us from passing a className manually though.

Reply:
Example of the PlayButton doing this as well

https://github.com/bbc/psammead/blob/latest/packages/components/psammead-play-button/src/index.jsx#L73

Reply:
I've removed the example since it's unnecessary, but as it was showing, when `styled` is used on a component
```
const StyledMostRead = styled(MostRead)`
  background-color: white;
`;
```
A className is generated and then passed in to it, in this case `MostRead`.

Reply:
Example from docs: https://styled-components.com/docs/basics#styling-any-component

Reply:
Thanks for the explanations @mmcculloch , maybe I'm confused but I think we don't need to pass a `className` to be able to style the component from somewhere else. 

There are few places in Simorgh and Psammead where we just import the component and then wrap it with `styled` where we want to expand/override its styles and from what I'm aware they don't have a `className`.

E.g. 
https://github.com/bbc/simorgh/blob/latest/src/app/containers/Headings/index.jsx#L16
https://github.com/bbc/simorgh/blob/latest/src/app/containers/RadioPageMain/index.jsx#L17
https://github.com/bbc/simorgh/blob/latest/src/app/containers/RadioPageMain/index.jsx#L17

Reply:
I've done some testing and now I understand the functionality of the `className` https://codesandbox.io/s/gracious-rhodes-lilfy?fontsize=14&hidenavigation=1&theme=dark

Basically, it is needed when you want to style any component which **has not** been defined using styled-components**, and probably those components which we extend its styles in our codebase at the moment, have been defined using `styled-components`, therefore my confusion.

Reply:
Yeah that makes sense. The others are defined by styled-components so automatically have this functionality. Especially Headline which is a fairly basic styled element.